src/components/country-labels-layer/country-labels-layer-component.jsx

```diff
@@ -63,7 +62,7 @@ const CountryLabelsLayerComponent = props => {
   useEffect(() => {
     let graphicLayer;
     let countryPinMarker;
-    if (layerReady && countryISO && !isCountryMode) {
+    if (layerReady && countryISO) {
```

Comment:
Maybe not entirely related with this PR but can you please check the Pin adding/removing logic? It is not working correctly
![image](https://user-images.githubusercontent.com/6906348/88630635-30847f80-d0b1-11ea-9d9d-60394f326447.png)